Documentation/NLog.config

```diff
@@ -1,138 +1,132 @@
+    <target xsi:type="null" name="null" formatMessage="false" />
   </targets>
   <rules>
     <!-- Avoid logging to incorrect folder before the logging initialization is done. If you want to see those logging messages, comment out this line, but your log file will be somewhere else. -->
     <logger name="*" minlevel="Trace" writeTo="null" final="true" />
-    
+
     <logger name="Stratis.Bitcoin.Features.Api.*" minlevel="Trace" writeTo="debugApiFile" />
-    
+
     <logger name="Stratis.Bitcoin.Features.BlockStore.*" minlevel="Trace" writeTo="debugBlockStoreFile" />
-    
+
     <logger name="Stratis.Bitcoin.Features.Consensus.*" minlevel="Trace" writeTo="debugConsensusFile" />
-    <logger name="Stratis.Bitcoin.Features.Consensus.CoinViews.*" minlevel="Trace" writeTo="debugCoinViewsFile" />   
-    <logger name="Stratis.Bitcoin.Features.Consensus.StakeValidator" minlevel="Trace" writeTo="debugConsensusValidationFile" />
-    <logger name="Stratis.Bitcoin.Features.Consensus.PosConsensusValidator" minlevel="Trace" writeTo="debugConsensusValidationFile" />
-    <logger name="Stratis.Bitcoin.Features.Consensus.PowConsensusValidator" minlevel="Trace" writeTo="debugConsensusValidationFile" />
-    <logger name="Stratis.Bitcoin.Features.Consensus.Rules.*" minlevel="Trace" writeTo="debugConsensusValidationFile" />
+	<logger name="Stratis.Bitcoin.Consensus.*" minlevel="Trace" writeTo="debugConsensusFile" />
+	
+	<logger name="Stratis.Bitcoin.Consensus.ChainedHeaderTree" minlevel="Trace" writeTo="debugCHTandCMFile" />
+	<logger name="Stratis.Bitcoin.Consensus.ConsensusManager" minlevel="Trace" writeTo="debugCHTandCMFile" />
+	
+    <logger name="Stratis.Bitcoin.Features.Consensus.CoinViews.*" minlevel="Trace" writeTo="debugCoinViewsFile" />
 
     <logger name="Stratis.Bitcoin.Features.Dns.*" minlevel="Trace" writeTo="debugDnsFile" />
-    
+
     <logger name="Stratis.Bitcoin.Features.LightWallet.*" minlevel="Trace" writeTo="debugLightWalletFile" />
-    
+
     <logger name="Stratis.Bitcoin.Features.MemoryPool.*" minlevel="Trace" writeTo="debugMemoryPoolFile" />
-    
+
     <logger name="Stratis.Bitcoin.Features.Miner.*" minlevel="Trace" writeTo="debugMinerFile" />
-    
+
     <logger name="Stratis.Bitcoin.Features.Notifications.*" minlevel="Trace" writeTo="debugNotificationsFile" />
-    
+
     <logger name="Stratis.Bitcoin.Features.RPC.*" minlevel="Trace" writeTo="debugRPCFile" />
-    
+
     <logger name="Stratis.Bitcoin.Features.Wallet.*" minlevel="Trace" writeTo="debugWalletFile" />
-    
+
     <logger name="Stratis.Bitcoin.Features.WatchOnlyWallet.*" minlevel="Trace" writeTo="debugWatchOnlyWalletFile" />
-    
-    <logger name="Stratis.Bitcoin.Base.*" minlevel="Trace" writeTo="debugBaseFile" />    
-    <logger name="Stratis.Bitcoin.Base.ChainHeadersBehavior" minlevel="Trace" writeTo="debugChainHeadersBehaviorFile" />
-    <logger name="Stratis.Bitcoin.Base.TimeSyncBehaviorState" minlevel="Trace" writeTo="debugTimeSyncFile" />
```

Comment:
Why to remove these two?

Reply:
ChainHeadersBehavior - now we don't have such class
TimeSyncBehaviorState- already covered in base. In the past 6 month we never needed logs explicitly for TimeSyncBehavior and in general it's already stable, no need to have separated target for it, it's ok for it to be in base log

Reply:
How do you know we never needed logs for time sync? You're making claim that you cannot backup. I would suggest to revert it back

Reply:
Fixed